### PR TITLE
Update tn035-using-multiple-resource-files-and-header-files-with-visu…

### PR DESCRIPTION
**Pull request by rtischer8277**
…al-cpp.md

Over the years I have read TN035 maybe a dozen times. Each time I have to (try) to parse the text, figure out what it really means, get past the fact that MS and users aren't going to fix it, then and only then can I solve my programming task at hand. 
Only this time, I worked on the text itself and made it clear(er) and unambiguous so at least I in the future can read TN035 and only use 6 minutes to understand the .RC file's Resource Includes dialog instead of a day or two.

### CODE REVIEW

**Comment by jborsecnik** [comment]
@TylerMSFT, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger19** [comment]
@rtischer8277 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@rtischer8277 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 1]
```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
...
 
-### Symbol Header File
+### Resource View's Resource Includes dialog
 
-By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). Using the **Resource Includes** command from the **View** menu in Visual C++, you can change the name of this header file by updating the Symbol Header File file in the **Set Includes** dialog box.
+#### Symbol Header File
 
-### Read-Only Symbol Directives
+By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). The **Symbol Header File:** section in the **Resource Includes** dialog in Visual C++, lets you change the name of this header file. Enter a new file name in the section's edit box.
```
Comment (AnjuDel) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:54]: Let's add that the Resource includes is accessible from right clicking the .rc file form the resource view

**Review by AnjuDel — CHANGES_REQUESTED** [review]

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 1 above)
Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:54]: Should `#include`d be back ticked?
[NOTE!] needs to be properly formatted. I don't know what that should be.
I added line #51 which addresses AnjuDel's comment. I included Resource View because Resource Includes can be accessed in at least three different ways: from the .rc file in Solution Explorer, from the VS ToolBar->View->Other Windows, and through accelerator keys. Not wanting to risk being too tedious, I left out discussion of Open With and the subsequent choice of viewing editor (JSON? XML? etc).

Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:54]: When #include is part of code, it should have back ticks. When it is part of UI, e.g. **Resource Includes**, no.
The format for notes is, and I'll code escape this so the GitHub formatter doesn't mangle it:
`> [!NOTE]
> Note text starts here`

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 2]
```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
...
+
+#### Compile-Time Directives
 
-Visual C++ also supports nesting of resource files, where one .RC file is #include'd within another. When you edit a given .RC file using Visual C++, any resources in the #include'd files are not visible. But when you compile the .RC file, the #include'd files are also compiled. Using the **Resource Includes** command from the **View** menu in Visual C++, you can specify any number of #include'd .RC files as Compile-Time Directives.
+Visual C++ also supports nesting of resource files, where one .RC file is #include'd within another. When you edit a given .RC file using Visual C++, any resources in the #include'd files are not visible. But when you compile the .RC file, the #include'd files are also compiled. The **Compile-Time Directives:** section in the **Resources Includes** dialog in Visual C++, lets you specify any number of #include'd .RC files as Compile-Time Directives.
```
Comment (rtischer8277) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:65]: Visual C++ also supports nesting of resource files, where one .RC file is `#include`'d within another. When you edit a given .RC file using Visual C++, any resources in the `#include`'d files are not visible. But when you compile the .RC file, the `#include`'d files are also compiled. The **Compile-Time Directives:** section in the **Resources Includes** dialog in Visual C++ lets you specify any number of `#include`'d .RC files as Compile-Time Directives.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 3]
```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
...
 
 Note what happens if you read into Visual C++ an .RC file that #include's another .RC file that is *not* specified as a Compile-Time Directive. This situation might arise when you bring to Visual C++ an .RC file that you had been previously maintaining manually with a text editor. When Visual C++ reads the #include'd .RC file, it merges the #include'd resources into the parent .RC file. When you save the parent .RC file, the #include statement, in effect, will be replaced by the #include'd resources. If you do not want this merge to happen, you should remove the #include statement from the parent .RC file *prior* to reading it into Visual C++; then using Visual C++, add back the same #include statement as a Compile-Time Directive.
```
Comment (rtischer8277) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:67]: Note what happens if you read into Visual C++ an .RC file that `#include`'s another .RC file that is *not* specified as a Compile-Time Directive. This situation might arise when you bring to Visual C++ an .RC file that you had been previously maintaining manually with a text editor. When Visual C++ reads the `#include`'d .RC file, it merges the `#include`'d resources into the parent .RC file. When you save the parent .RC file, the `#include` statement, in effect, will be replaced by the `#include`'d resources. If you do not want this merge to happen, you should remove the `#include` statement from the parent .RC file *prior* to reading it into Visual C++; then using Visual C++, add back the same `#include` statement as a Compile-Time Directive.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 1 above)
Comment (rtischer8277) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:54]: By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). The **Symbol Header File:** section in the **Resource Includes** dialog in Visual C++ lets you change the name of this header file. Enter a new file name in the section's edit box.

**Comment by TylerMSFT** [comment]
@AnjuDel , if you are good with this, I'm fine with merging it.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 4]
```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
 
 Visual C++ manages a single .RC resource file and a corresponding .H header file as a tightly coupled pair of files. When you edit and save resources in an .RC file, you indirectly edit and save symbols in the corresponding .H file. Although you can open and edit multiple .RC files at a time (using Visual C++'s MDI user interface) for any given .RC file you indirectly edit exactly one corresponding header file.
 
-### Symbol Header File
+### Resource View's Resource Includes dialog
```
Comment (AnjuDel) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:50]:
```suggestion
### Resource View's Resource Includes dialog

You can access the **Resource Includes** by going to the **Resource View** then right clicking the .rc file and selecting **Resource Includes**
```

**Review by AnjuDel — CHANGES_REQUESTED** [review]

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 5]
```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
...
 
-By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). Using the **Resource Includes** command from the **View** menu in Visual C++, you can change the name of this header file by updating the Symbol Header File file in the **Set Includes** dialog box.
+#### Symbol Header File
 
-### Read-Only Symbol Directives
+By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). The **Symbol Header File:** section in the **Resource Includes** dialog in Visual C++, lets you change the name of this header file. Enter a new file name in the section's edit box.
+
+> [!NOTE]
+> resource files not located in the same directory as the .RC file must prepend a relative path with escaped-'\\' to be read properly.
```
Comment (rtischer8277) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:57]: resource files not located in the same directory as the .RC file must prepend a relative path with respect to the project directory to be read properly.

**Comment by rtischer8277** [comment]
Sorry for the late change, but my Note comment had to be changed. I excluded escaped-'\\' and added 'with respect to the project directory'. The escaped text works in certain places if you are manually editing the .rc file and that is not the focus of TN035.

**Comment by PRMerger4** [comment]
@TylerMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 5 above)
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:59]: Can you remove "to be read properly" We try to be succinct and if we are telling them to do something, it's so it will work properly.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 6]
```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
...
+#### Read-Only Symbol Directives
 
-### Compile-Time Directives
+Although Visual C++ only edits one header file for any given .RC file, Visual C++ supports references to symbols defined in additional read-only header files. The **Read-Only symbol Directives:** section in the **Resource Includes** dialog in Visual C++, lets you specify any number of additional read-only header files as Read-Only Symbol Directives. The "read-only" restriction means that when you add a new resource in the .RC file, you can use a symbol defined in the read-only header file; but if you delete the resource, the symbol still remains defined in the read-only header file. You cannot change the numeric value assigned to a read-only symbol.
```
[the remark this answers is not in the document]
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:63]: Can this be: "In the Visual Studio **Resources Includes** dialog, in the **Resource Includes** section, specify additional read-only heade3r files as **Read-only symbol directives**. The *read-only* restriction means that the symbol defined for a new resource in the `.RC` file remains even if you delete the resource. You can't change the numeric value assigned to a read-only symbol.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 3 above)
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: How about:
At compile time, Visual Studio merges `#include` statements in a child `.RC` file into the parent `.RC` file that `#include`s it. If you don't want this behavior, remove the `#include` statement from the parent `.RC` file *prior* to adding the file to Visual Studio. Then, in Visual Studio, add the `#include` statement as a **Compile-Time Directive**.

Even with this change to try to get to the heart of the matter, I'm still not sure what exactly happens. Are you saying that VS rewrites the parent .RC file to have all the #include statements from the child?

**Review by TylerMSFT — CHANGES_REQUESTED** [review]
Left some comments.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 3 above)
Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: > ...it merges the #include'd resources into the parent .RC file.

How about:
Before compile time, resources like *Ribbon, Menu, Bitmap, Dialog*, etc. are visually edited resulting in changes to, for example, screen positions and IDs and target objects. The `ribbon.mfcribbon-ms` xml file organizes ID, file name changes and object relations for the ribbon. 

At compile time, which occurs when the Visual Studio program is built, these changed resource files become RC Compiler input and get compiled with other **Compile-time directives** like `afxres.rc` and `afxprint.rc` as well as **Read-only symbol directives** like `resource.h` and `myResource.h`. This RC Compiling produces a new `.rc` file. It is this *.rc* file that serves as object data for the rest of the build.

Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: Hi @rtischer8277 ,  now we seem to have lost the suggestion you were making to devs: "If you do not want this merge to happen..."  Should that be part of this?  You can also remove redundancy such as explaining what compile time is.

Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: Are you on-board with my reasoning for this section?

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:63]: Neither:

> The "read-only" restriction means that when you add a new resource in the .RC file, you can use a symbol defined in the read-only header file; but if you delete the resource, the symbol still remains defined in the read-only header file. You cannot change the numeric value assigned to a read-only symbol.

nor

> The read-only restriction means that the symbol defined for a new resource in the .RC file remains even if you delete the resource. You can't change the numeric value assigned to a read-only symbol.

are right from my view. The problem for me is, even though, fx, `resource.h` header file is changed by the editors, it is still merely input to the `RC` Compiler at program build time. Besides, there is no *read-only* restriction in the header files. It is easy to change the numeric value of a symbol with any text editor, as long as you modify the counters at the bottom accordingly. And, of course, the resource editors change the header IDs at any time. So, it's not really good advice. My advice is to just remove the two sentences.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 1 above)
Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:56]: I would like to re-add my note about relative path-names since it is valid and necessary for the `RC` Compiler to find the header files:

> [!NOTE]
If the header file is not in the same directory as the project, pre-pend the new `MYRESOURCE.H` header name in the **Symbol Header File** edit box with a relative path.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 5 above)
Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:59]: Yes. Also, I adjusted the Note text so it is specific to the section at hand. The text becomes:

[!NOTE]
If the compile-time directive file is not in the same directory as the project, pre-pend the new `MYDIRECTIVES.RC`  file name with a relative path.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 1 above)
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:56]: Minor edits: "If the header file isn't in the same directory as the project, in **Symbol Header File**, prepend the new `MYRESOURCE.H` header name with a relative path.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:63]: agree

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 3 above)
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: This is too big for a note. It should be its own section. It's also hard to scan.  Can it be broken down into bullets or a table for special cases and the corresponding resolution?

What does this mean: If that occurs, a previous .rc file will be needed to bring RC Compiling back to a functioning state

Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: @rtischer8277, thank you for hanging in on these changes. I think you are making an important contribution to the article.
Once we get through the main ideas of your changes, I think rather than do an editing pass back and forth in this PR, I'll have you check in and later I'll go through and do the all the Microsoft editing things like using correct branding, style updates, etc. Good with you?

Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: That's what I proposed, I thought :-)

Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: I would like to take a stab at re-writing the *"Note what happens if you read into Visual C++ an .RC file..."* paragraph. I have collected all the edits and consolidated them in the following text:

### **Resource Includes** directives ( non-primary `.rc` script file(s) not added to Visual Studio project)
> [!NOTE]
You can access the **Resource Includes** by going to the **Resource View** then right clicking the `.rc` file and selecting **Resource Includes**.

####Symbol Header File
By default, Visual C++ always names the corresponding header file `RESOURCE.H`, regardless of the name of the resource file (e.g., `MYAPP.RC`). Using the **Resource Includes** command from the **Resource View** menu in Visual C++ lets you can change the name of this header file by updating the **Symbol Header File** file name in the **Set Includes** edit box.
> [!NOTE]
If the header file isn't in the same directory as the project, in **Symbol Header File**, prepend the new `MYRESOURCE.H` header name with a relative path.

####**Read-only symbol directives**
Although Visual C++ only edits one header file for any given .RC file, Visual C++ supports references to symbols defined in additional read-only header files. The **Read-only symbol directives** section in the **Resource Includes** dialog in Visual C++ lets you specify any number of additional header files.

####**Compile-time directives**
Other non-primary `.rc` script files may be added directly to your primary `.rc` script file using this section. Their effect is the following:

Before compile time, resources like *Ribbon, Menu, Bitmap, Dialog*, etc. are visually edited resulting in changes to, for example, screen positions and IDs and target objects. While the `.rc` file organizes IDs, file name changes and their paths, the `ribbon.mfcribbon-ms` xml file organizes IDs and object relations to each other for the ribbon. Resource editing changes affect this file.

At compile time, which occurs when the Visual Studio program is built, these changed resource files become `RC` Compiler input and get compiled with other **Compile-time directives** like `afxres.rc` and `afxprint.rc` as well as **Read-only symbol directives** like `resource.h` and `myResource.h`. This `RC` Compiling produces a new `.rc` file. It is this `.rc` file that serves as object data for the rest of the build.

Visual Studio C++ compiles output a new `.rc` file reflecting the three kinds of **Resource Includes** information (**Symbol header file**, **Read-only symbol directives**, and **Compile-time directives**). `#include` directives appear in `TEXTINCLUDE` resources. The `TEXTINCLUDE` resources, an implementation detail that you do not normally need to deal with, are explained in [How Visual C++ Manages Set Includes Information](https://docs.microsoft.com/en-us/cpp/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp?view=msvc-170#_mfcnotes_tn035_set_includes).

### **Resource Includes** directives (non-primary `.rc` script file(s) added to Visual Studio project)
Another way to deal with other `.rc` script files is to add them directly to your Visual Studio project's solution. Your other `.rc` files will then appear as separate menu items in your solution's *Resource View*. One advantage to this method is Visual Studio manages access to the file's location, so there is no need for relative paths. Another advantage is, you can use any or all of Visual Studio's resource editors to manipulate its resource contents. There is, however, a major disadvantage. When using the resource editors to make changes in a non-primary `.rc` file, the `RC` Compiler is not able to save the changes and shows the *Cannot save file* message. Once visual `.rc` script editing is in this state, it is necessary to close the Visual Studio project and reopen it in order to discard the changes. To not get in this resource editing state, do all your resource editing in your main `.rc` file which is always able to save your changes. Once the visual editing changes are complete, save them, and then copy manually any changed `.rc` script definition blocks into your destination `.rc` file(s). The sum of the script definitions must, however, always be mutually exclusive at build time. To achieve that, temporarily comment out the definition blocks in the non-primary `.rc` files, that are duplicates.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md [hunk 7]
```diff
@@ -47,17 +47,24 @@ This note is structured to answer the above questions as follows:
 
 Visual C++ manages a single .RC resource file and a corresponding .H header file as a tightly coupled pair of files. When you edit and save resources in an .RC file, you indirectly edit and save symbols in the corresponding .H file. Although you can open and edit multiple .RC files at a time (using Visual C++'s MDI user interface) for any given .RC file you indirectly edit exactly one corresponding header file.
 
-### Symbol Header File
+### Resource View's Resource Includes dialog
```
Comment (AnjuDel) [inline docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:50]:
```suggestion
### Resource View's Resource Includes dialog

You can access the **Resource Includes** by going to the **Resource View** then right clicking the .rc file and selecting **Resource Includes**
```

**Review by AnjuDel — CHANGES_REQUESTED** [review]

**Comment by TylerMSFT** [comment]
Hi @rtischer8277, I'd like to get your work incorporated into the docs. There was a comment from Anju. After ou take a look at that, do you have any other things we need to finish off before we get this in? I have multiple projects going on at the moment and apologize if I've dropped the ball on an outstanding issue you need me to follow up on here.

File: docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md (hunk 3 above)
Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: I like this version. I think it is only missing some context about why you want to read this note. I'm thinking of an introduction along the lines of "you want to do x, here's how" or "you have problem x, here's how" or "you've wondered how this machinery works in detail, here's how" that sets up why we are reading this text.

Comment (rtischer8277) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: I saw Anju's comment, but didn't understand it. Maybe with more explanation I could see where it may apply to the current draft. I suggest that the comment be applied to another PR in the future where it can be fully discussed.

I agree with your context intro and here is some introductory text. If the text is too wordy, let me know and I'll cut it down, or you can. I am also setting *Conversation Resolved* and am ready to check it in:

I suggest that we proceed towards check-in with this PR.

Here is the suggested Introduction to the draft:

**Introduction to cross-project `.rc` script file re-use**
What marks production code from experimental code is the necessity for eliminating code duplication. To the degree present, duplicate code renders production code project(s) un-maintainable over time. To combat this tendency, C++ has always observed the one-definition rule (ODR) and no changes are allowed into ISO C++ that do not adhere to ODR. MFC C++ can be written to observe ODR as well, and therefore can maintain cross-project maintainability thanks to ISO C++. But Visual Studio C++'s `.rc` script file is not C++ and therefore requires special attention to achieve cross-project ODR status. This means breaking up `.rc` files into separate scripts that make sense for your set of projects. Below is a description of how this is done:

Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: Thanks, @rtischer8277 . I pinged Anju to respond to the issue he raised. I think the ISO C++ connection is unnecessary (and a slightly stretched connection in this context). People will understand that you don't want duplicate copies of stuff. I'd just say you don't want duplicate copies of things in your resources, here's how to do it.

Comment (TylerMSFT) [reply docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md:69]: Great

**Comment by TylerMSFT** [comment]
@AnjuDel , can you close with @rtischer8277 on the issue you had so we can get this merged?

**Review by TylerMSFT — APPROVED** [review]
@rtischer8277 , I talked to Anju and we're going to go ahead and incorporate your changes.
Thank you for taking the time to contribute to the docs!

**Comment by TylerMSFT** [comment]
#sign-off
